packages/eslint-config-sheriff/package.json

```
@@ -46,14 +46,16 @@
   "scripts": {
     "preinstall": "npx only-allow pnpm",
     "clean": "rm -rf .turbo dist node_modules/.cache",
-    "re-bundle-types": "dts-bundle-generator -out-file=dist/index.d.ts ./dist/index.d.ts",
+    "re-bundle-types": "dts-bundle-generator --config dts-config.json",
+    "are-the-types-wrong": "bash -c 'attw $(pnpm pack) --ignore-rules cjs-resolves-to-esm no-resolution && rm eslint-config-sheriff-*.tgz'",
```

Before this gets merged, we can simplify this because attw released v0.17, which adds profiles.

```suggestion
    "are-the-types-wrong": "bash -c 'attw $(pnpm pack) --profile=esm-only && rm eslint-config-sheriff-*.tgz'",
```